dev/com.ibm.ws.logging/src/com/ibm/ws/logging/collector/CollectorJsonHelpers.java

```diff
@@ -462,6 +461,15 @@ private static boolean verifyFloatValue(String s) {
         return isValid;
     }
 
+    public static boolean addPairBasedOnType(String extKey) {
+        if (extKey.endsWith(CollectorJsonHelpers.INT_SUFFIX) || extKey.endsWith(CollectorJsonHelpers.FLOAT_SUFFIX)
+            || extKey.endsWith(CollectorJsonHelpers.BOOL_SUFFIX)) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
     protected static boolean checkIfExtIsQuoteless(String extKey) {
```

Comment:
this method can probably be removed since type-specific KVPs handle this for you